teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -157,34 +257,32 @@ Security! =)
 We'll also need this feature for Tekton to achieve [SLSA 3](https://slsa.dev/levels), which requires Non-falsifiable Provenance.
 
 ### Requirements
-* SPIRE is configured to only issue SVIDs to the Tekton controller via a [k8s Workload Attestor](https://github.com/spiffe/spire/blob/main/doc/plugin_agent_workloadattestor_k8s.md)
+* SPIRE is configured to only issue SVIDs to the tekton-pipelines-controller via a [k8s Workload Attestor](https://github.com/spiffe/spire/blob/main/doc/plugin_agent_workloadattestor_k8s.md)
 
 ### Goals
 
 * Results can be verified as non-falsifiabe
 * A TaskRun yaml can be verified as non-falsifiable
```

Comment:
this is not updated as part of this PR but its a little confusing (sorry to point it out at this point). What does `taskRun` yaml mean here? My understanding is you are referencing to a taskRun resource (CRD), right? 🤔